R/render.r

```diff
@@ -432,7 +432,7 @@ data_deps <- function(pkg, depth) {
   # Function needed for indicating where that deps folder is compared to here
   transform_path <- function(x) {
 
-    x <- gsub(pkg$dst_path, "", x)
+    x <- sub(htmltools::urlEncodePath(pkg$dst_path), "", htmltools::urlEncodePath(x))
```

Comment:
Can you explain why this works?

Reply:
Which also makes me realize I was using the function one time too many.